<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Socket.IO Chat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sendBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listView">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>41</y>
      <width>351</width>
      <height>431</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="nickNameEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Your nick name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>7</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>485</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Type here...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>366</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>loginBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>LoginClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messageEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnMessageReturn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SendBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messageEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>TypingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SendBtnClicked(bool)</slot>
  <slot>TypingChanged()</slot>
  <slot>LoginClicked()</slot>
  <slot>OnMessageReturn()</slot>
  <slot>SendBtnClicked()</slot>
 </slots>
</ui>
